@ornikar/eslint-config-react/tests/no-unescaped-entities.js

```diff
@@ -0,0 +1,15 @@
+function App() {
+  return null;
+}
+
+export function NoUnescapedEntities() {
+  return (
+    <App>
+      {/* eslint-disable-next-line react/no-unescaped-entities */}
+      <p>A paragraph with 'quote' is incorrect</p>
+      <p>A paragraph with "quote" is correct</p>
+      <div> {'<'} </div>
+      <div> &gt; </div>
```

Comment:
on veut jamais ca

Reply:
C'est les 2 div que l'on veut jamais ? Ou l'exemple qui est mal approprié ?

Reply:
avoir `&gt;`

Reply:
https://ornikar.atlassian.net/wiki/spaces/TECH/pages/982712688/Internationalisation+Guidelines+react-intl+and+more#Non-breaking-spaces-and-other-html-escapes